debian/copyright

```diff
@@ -1,28 +1,57 @@
-This package was debianized by Michael Prokop <mika@grml.org> on
-Thu, 28 Jul 2005 08:50:04 +0200.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://github.com/grml/grml-hwinfo/
+
+Files: *
+Copyright: 2003-2025 Michael Prokop <mika@grml.org>
+License: GPL-2
+
+Files: grml-hwinfo
+License: GPL-2
+Copyright: 2007 Alexander Wirt <formorer@debian.org>
+           2011 Chris Hofstaedtler <zeha@debian.org>
+           2024 Christopher Bock <christopher@bocki.com>
+           2023, 2024 Darshaka Pathirana <dpat@syn-net.org>
+           2003-2025 Michael Prokop <mika@grml.org>
+           2011 Richard Hartmann <richih.mailinglist@gmail.com>
+           2010 Ulrich Dangel <uli@spamt.net>
+
+Files: sysdump
+License: GPL-2+
+Copyright: 2005 David Schmitt <david@schmitt.edv-bus.at>
+           2018 Darshaka Pathirana <dpat@syn-net.org>
+           2006, 2017, 2021 Michael Prokop <mika@grml.org>
+
+Files: debian/*
+License: GPL-2
```

Comment:
we had GPL-2+ in debian/copyright, so let's keep it like that?

Reply:
i figured its pointless to have GPL-2+ for debian/*, when the rest is GPl-2 :/